torchvision/prototype/transforms/_container.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Optional, Dict
+
+import torch
+
+from ._transform import Transform
+
+
+class _WrapperTransform(Transform):
```

Comment:
We should consider the option of simplifying the inheritance patterns. If not necessary `_WrapperTransform` and `_MultiTransform` can be eliminated.